events/equeue/equeue_mbed.cpp

```diff
@@ -131,7 +131,7 @@ static void equeue_sema_timeout(equeue_sema_t *s) {
 bool equeue_sema_wait(equeue_sema_t *s, int ms) {
     int signal = 0;
     Timeout timeout;
-    if (ms > 0) {
+    if (ms >= 0) {
```

Comment:
Hmm, that's a good point. Do you think we should clamp it to 1 millisecond to be safe?

It does look like timer events in the past are handled correctly (in common code):
- during insertion: https://github.com/ARMmbed/mbed-os/blob/master/hal/mbed_ticker_api.c#L70
- during the irq handler: https://github.com/ARMmbed/mbed-os/blob/master/hal/mbed_ticker_api.c#L37-L46